readme.md

```diff
@@ -254,6 +255,8 @@ _Alphabet of Japanese language._
   - [imiwa?](http://www.imiwaapp.com/) - iOS Japanese dictionary app.:iphone:
   - [Tagaini Jisho](https://www.tagaini.net/) - Open-source Japanese dictionary and kanji lookup tool for Windows, MacOS X and Linux.
   - [Jiten](https://play.google.com/store/apps/details?id=dev.obfusk.jiten) - Android Japanese dictionary and kanji lookup app, also has web version.:iphone:
+  - [Japanese](https://apps.apple.com/it/app/japanese/id290664053) - Dictionary, stroke order and flashcards for iOS :iphone:
```

Comment:
japaneseapp.com
which is the official website, and offer links to both android and ios version